src/EditorFeatures/Core/Implementation/Workspaces/EditorTextFactoryService.cs

```diff
@@ -40,6 +40,11 @@ internal class EditorTextFactoryService : ITextFactoryService
 
         private static readonly Encoding s_throwingUtf8Encoding = new UTF8Encoding(encoderShouldEmitUTF8Identifier: false, throwOnInvalidBytes: true);
 
+        /// <summary>
+        /// The default algorithm used by <see cref="SourceText"/> is <see cref="SourceHashAlgorithm.Sha1"/>.
+        /// </summary>
+        public SourceHashAlgorithm ChecksumAlgorithm => SourceHashAlgorithm.Sha1;
```

Comment:
i forget, but are we allowed to use sha1?  i thought that was deprecated/banned for new code.

Reply:
➡️ This is the only valid implementation of this property without making a behavior change outside the scope of this pull request.

Reply:
sgtm.